stdlib/public/Platform/Platform.swift

```diff
@@ -187,6 +209,7 @@ public func open(
   return _swift_Platform_open(path, oflag, 0)
 }
 
+#if !os(Windows) || CYGWIN
 public func open(
   _ path: UnsafePointer<CChar>,
   _ oflag: Int32,
```

Comment:
Sorry, I meant, that because it calls `openat`.  Now, arguably that is a bug, and we should fix that :-).